### DISCUSSION TITLE
Recipe terminated with fatal error: spawn xelatex ENOENT.

### DISCUSSION THREAD

**Discussion by theslyprofessor** [James-Yu/LaTeX-Workshop#3129] [Q&A]
In order to fix this, I know there's something I know I need to do with the path, but can someone tell me **exactly where/how** to fix this? I'm pretty sure I have to set some env value, but I don't know where to look for it, and what value to input. I've read through documentation, but I haven't been able to make sense of it. I'm looking for something that I can just cut and paste.

Thanks,
Nakul

**Comment by tamuratak** [comment]
See

- https://github.com/James-Yu/LaTeX-Workshop/wiki/Install#setting-path-environment-variable
- https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ#environment-variables-in-bashrc-dont-take-effect-with-vs-code-remote

**Comment by tamuratak** [comment]
Please fill out the [issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md) next time.
